pvlib/temperature.py

```diff
@@ -650,19 +653,52 @@ def ross(poa_global, temp_air, noct):
 
     .. math::
 
-        T_{C} = T_{a} + \frac{NOCT - 20}{80} S
+        T_{C} = T_{a} + \frac{NOCT - 20}{80} S = T_{a} + k × S
 
     where :math:`S` is the plane of array irradiance in :math:`mW/{cm}^2`.
     This function expects irradiance in :math:`W/m^2`.
 
+    Reference values for k are provided in [2]_, covering different types
+    of mounting and module structure.
+
+    +---------------------+-----------+
+    | Mounting            | :math:`k` |
+    +=====================+===========+
+    | well_cooled         | 0.02      |
+    +---------------------+-----------+
+    | free_standing       | 0.0208    |
+    +---------------------+-----------+
+    | flat_on_roof        | 0.026     |
+    +---------------------+-----------+
+    | not_so_well_cooled  | 0.0342    |
+    +---------------------+-----------+
+    | transparent_pv      | 0.0455    |
+    +---------------------+-----------+
+    | facade_integrated   | 0.0538    |
+    +---------------------+-----------+
+    | on_sloped_roof      | 0.0563    |
+    +---------------------+-----------+
+
     References
     ----------
     .. [1] Ross, R. G. Jr., (1981). "Design Techniques for Flat-Plate
        Photovoltaic Arrays". 15th IEEE Photovoltaic Specialist Conference,
        Orlando, FL.
+    .. [2] E. Skoplaki and J. A. Palyvos, “Operating temperature of
+       photovoltaic modules: A survey of pertinent correlations,” Renewable
+       Energy, vol. 34, no. 1, pp. 23–29, Jan. 2009,
+       :doi:`10.1016/j.renene.2008.04.009`
     '''
-    # factor of 0.1 converts irradiance from W/m2 to mW/cm2
-    return temp_air + (noct - 20.) / 80. * poa_global * 0.1
+    if (noct is None) & (k is None):
+        raise ValueError("noct or k need to be provided as numeric input.")
+    elif (noct is not None) & (k is not None):
+        raise ValueError("Provide only noct or k, not both.")
+    elif k is None:
+        # factor of 0.1 converts irradiance from W/m2 to mW/cm2
+        return temp_air + (noct - 20.) / 80. * poa_global * 0.1
```

Comment:
When the docstring is edited in #2519 please adjust this equation to remove the factor 0.1